lib/g_searcher/search_results/search_result.ex

```diff
@@ -0,0 +1,51 @@
+defmodule GSearcher.SearchResults.SearchResult do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "search_results" do
+    field :search_term, :string
+    field :total_number_results, :integer
+    field :number_of_results_on_page, :integer
+    field :all_urls, {:array, :string}
+    field :html_cache, :string
+
+    # Top positioned advertisers
+    field :number_of_top_advertisers, :integer
+    field :top_advertiser_urls, {:array, :string}
+
+    # All advertisers
+    field :total_number_of_advertisers, :integer
+    field :advertiser_urls, {:array, :string}
+
+    timestamps()
+  end
+
+  def create_keyword_changeset(search_result \\ %__MODULE__{}, attrs) do
+    search_result
+    |> cast(attrs, [:search_term])
+    |> validate_required(:search_term)
+  end
+
+  def create_search_result_changeset(search_result \\ %__MODULE__{}, attrs) do
```

Comment:
I guess we don't need this one yet, in case we would like to keep it here, I think the name should be `update_...` instead of `create_..` what do you think @bterone as we will update the search result from Google later on

Reply:
Ah yes, you're right, I can remove this here. I'll change this to `update_...` in a future PR. Thanks for catching this! 👍